Jenkinsfile

```diff
@@ -23,7 +23,7 @@ pipeline {
       steps {
         checkout scm
         sh "git submodule update --jobs 8 --init --recursive"
-        sh "git clone -b v1.3.1 git@github0.xmos.com:xmos-int/xtagctl.git"
+        sh "git clone -b v1.5.0 git@github0.xmos.com:xmos-int/xtagctl.git"
```

Comment:
If we're going to specify the xtagctl version in a Jenkinsfile, shouldn't we make it a parameter?

Reply:
Agreed that no change is needed.